oktoberfest/data/spectra.py

```diff
@@ -229,7 +276,23 @@ def from_csv(cls: Type[SpectraT], input_file: Union[str, Path]) -> SpectraT:
         :return: a spectra instance
         """
         input_file = str(input_file)
-        spectra = cls()
-        spectra.spectra_data = pd.read_csv(input_file)
+        all_columns = pd.read_csv(input_file)
+        size = (all_columns.shape[0], 174)
+        spectra = cls(size)
+        spectra.add_columns(all_columns)
 
         return spectra
+
+    def convert_to_df(self) -> pd.DataFrame:
```

Comment:
maybe also have a look at this here: https://anndata.readthedocs.io/en/latest/generated/anndata.AnnData.to_df.html

Reply:
This function only does the following: `The data matrix .X is returned as pd.df, where obs_names initializes the index, and var_names the columns.` So it doesnt save the obs columns and the other layers. I think we can keep this function for now.